force-app/main/default/lwc/wireGetObjectInfo/wireGetObjectInfo.js

```diff
@@ -1,20 +1,14 @@
-import { LightningElement, wire, track } from 'lwc';
+import { LightningElement, wire } from 'lwc';
 import { getObjectInfo } from 'lightning/uiObjectInfoApi';
 
 export default class WireGetObjectInfo extends LightningElement {
-    objectApiNameInputValue = 'Account';
     objectApiName;
-    @track objectInfo;
 
     @wire(getObjectInfo, { objectApiName: '$objectApiName' })
     objectInfo;
 
-    objectNameChange(event) {
```

Comment:
No need to collect the value as it's entered. We can get it in the button click handler.